app/src/main/java/org/oppia/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -488,6 +575,24 @@ class StatePlayerRecyclerViewAssembler private constructor(
           fragment as SubmitNavigationButtonListener
         )
       }
+      doesMostRecentInteractionRequireExplicitSubmission(rightPendingItemList) &&
+        playerFeatureSet.interactionSupport && shouldSplit -> {
+        val canSubmitAnswer = checkNotNull(this.canSubmitAnswer) {
+          "Expected non-null submit answer observable for submit button when interaction support " +
+            "is enabled"
+        }
+        rightPendingItemList += SubmitButtonViewModel(
+          canSubmitAnswer,
+          /* hasPreviousButton= */ false,
+          previousNavigationButtonListener,
+          fragment as SubmitNavigationButtonListener
+        )
+        if (hasPreviousButton) {
```

Comment:
Is this correct? Do we want the previous button to be added to the right list?

Reply:
It isn't being added to the right list (`extraInteractionPendingItemList`) here, it is being added to the left one (`conversationPendingItemList`).  
```
extraInteractionPendingItemList += SubmitButtonViewModel(
          canSubmitAnswer,
          /* hasPreviousButton= */ false,
          previousNavigationButtonListener,
          fragment as SubmitNavigationButtonListener
        )
        if (hasPreviousButton) {
          conversationPendingItemList += PreviousButtonViewModel(
            previousNavigationButtonListener
          )
        }
```

Reply:
Resolving this on behalf of Ben. Also, this has been clarified with Chantel and as per initial conversation, the prev button is on right, but if it changes in future, we will file an issue and fix it.